unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapSearchController.cs

```diff
@@ -40,14 +86,14 @@ private void OnSearchedText(string searchText)
         }
 
         AddToPreviousSearch(searchText);
-
         searchCts = searchCts.SafeRestart();
         SearchAndDisplay(searchText, searchCts).Forget();
     }
 
     private async UniTaskVoid SearchAndDisplay(string searchText, CancellationTokenSource cts)
     {
         (IReadOnlyList<IHotScenesController.PlaceInfo> places, int total) searchPlaces = await placesAPIService.SearchPlaces(searchText, 0, 5, cts.Token);
+        exploreV2Analytics.SendSearchPlaces(searchText, searchPlaces.places.Select(p=>Utils.ConvertStringToVector(p.base_position)).ToArray(), searchPlaces.places.Select(p=>p.id).ToArray(), ActionSource.FromNavmap);
```

Comment:
search could be used often, so maybe it would be better to avoid LINQ due to its memory allocations (convert to foreach or for loop)